docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -119,17 +130,17 @@ Examples
 
 .. code-block:: cppkokkos
 
```

Comment:
Are the examples below copy - paste-able into Compiler Explorer?  If not, please make them simpatico with Compiler Explorer.